### DISCUSSION TITLE
Disable hover

### DISCUSSION THREAD

**Discussion by devopg** [saadeghi/daisyui#1511] [Q&A] [answered]
Hello i dont understand can i easy disable hover styles on button and it should work like btn?
like this
`<button class="btn btn-primary hover:none">Button</button>`

**Comment by saadeghi** [comment, marked as answer]
There's no `hover:none` class in Tailwind.  
A primary button has `primary` color by default and it changes to `primary-focus` color on hover. So you can do something like this to change the hover color back to what it was:
```html
<button class="btn btn-primary hover:bg-primary">Button</button>
```
